android/app/build.gradle

```diff
@@ -227,7 +230,11 @@ android {
 
         // Bugsnag integration
         manifestPlaceholders = [bugsnagReleaseStage: getDefaultBugsnagReleaseStage(myVersionName)]
+
+        // React Native Config
+        resValue "string", "build_config_package", "com.mapeo"
```

Comment:
following the step described here: https://github.com/lugg/react-native-config?tab=readme-ov-file#advanced-android-setup

Hoping I read it correctly and did the right thing, but not entirely sure tbh. Figured it was a relevant step because we do specify different suffixes depending on the build and command

Reply:
this step i skipped when I tried this library earlier. So this must be necesssary...